libs/agno/agno/tools/docker.py

```diff
@@ -0,0 +1,594 @@
+import json
+from typing import Dict, Optional
+
+from agno.tools import Toolkit
+from agno.utils.log import logger
+
+try:
+    import docker
+    from docker.errors import DockerException, ImageNotFound
+except ImportError:
+    raise ImportError("`docker` not installed. Please install using `pip install docker`")
+
+
+class DockerTools(Toolkit):
+    def __init__(
+        self,
+        enable_container_management: bool = True,
+        enable_image_management: bool = True,
+        enable_volume_management: bool = True,
+        enable_network_management: bool = True,
```

Comment:
This is a LOT of tools by default. Maybe only container management should be true by default?